test/quantization/test_quantize_script.py

```diff
@@ -1163,9 +1163,9 @@ class TestQuantizeScriptPTSQOps(JitTestCase):
     """ Test graph mode post training static quantization works
     for individual ops end to end.
     """
-    def _test_op_impl(self, SingleOpModule, data, quantized_op):
+    def _test_op_impl(self, SingleOpModule, data, quantized_op, **kwargs):
```

Comment:
can you change this to take a module instance? then you can initialize the module before passing into this function.